recipes/brewer2mpl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.4.1" %}
+
+package:
+  name: brewer2mpl
+  version: {{ version }}
+
+source:
+  fn: brewer2mpl-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/b/brewer2mpl/brewer2mpl-{{ version }}.tar.gz
+  md5: f8cd1fbb9f5d836a3a095b1ca9d58fc2
+
+build:
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - brewer2mpl
+    - brewer2mpl.wesanderson
+
+  requires:
+    - setuptools
+
+about:
+  home: https://github.com/jiffyclub/brewer2mpl/wiki
+  license: MIT License
```

Comment:
Doesn't it though ( https://github.com/conda-forge/conda-smithy/pull/93 )? 😉  We have tried this before, but decided to leave a lot of wiggle room for people, which has made it hard to add a good addition to the linter for licenses. Though I know @ocefpaf was thinking about giving this another try at one point.